### PR TITLE
Update Razor packages

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Includes bug fixes for Razor formatting

@NTaylorMullen @ryanbrandenburg @TanayParikh

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,10 +11,11 @@
 * Renaming symbol fails within a file that had recently been renamed without saving changes.
   * As a workaround, make an edit within the file before using Rename Symbol.
 
-## 1.23.3 (Not yet released)
+## 1.23.4 (Not yet released)
```
Comment (NTaylorMullen) [inline CHANGELOG.md:14]: Why 4?

**Review by NTaylorMullen — APPROVED** [review]

**Review by TanayParikh — APPROVED** [review]
LGTM other than the version change Taylor pointed out.

File: CHANGELOG.md (hunk 1 above)
Comment (ajaybhargavb) [reply CHANGELOG.md:14]: I thought releases once created cannot be modified.

Comment (TanayParikh) [reply CHANGELOG.md:14]: I think we update the pre-release beta version, without any changes here.
https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.23.3-beta1

**Comment by ajaybhargavb** [comment]
Updated. Also verified everything works fine locally.
@JoeRobich could you update the release?

**Comment by TanayParikh** [comment]
@ajaybhargavb CIs' failing, looks like it's expecting changes in the `package.lock.json`.

**Comment by ajaybhargavb** [comment]
cc @nohwnd

**Review by JoeRobich — APPROVED** [review]

**Comment by JoeRobich** [comment]
@ajaybhargavb New beta https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.23.3-beta2

**Pull request merged**